build.gradle

```diff
@@ -233,18 +233,23 @@ subprojects {
         ballerinaStdLibs "org.ballerinalang:task-ballerina:${stdlibTaskVersion}"
         ballerinaStdLibs "org.ballerinalang:reflect-ballerina:${stdlibReflectVersion}"
     }
+}
 
-    release {
-        // Disable check snapshots temporarily
-        buildTasks = []
-        failOnSnapshotDependencies = false
-        failOnCommitNeeded = false
-        versionPropertyFile = 'gradle.properties'
-        tagTemplate = 'v${version}'
-        git {
-            // To release from any branch
-            requireBranch = null
-        }
+def moduleVersion = project.version
+if (moduleVersion.indexOf('-') != -1) {
+    moduleVersion = moduleVersion.substring(0, moduleVersion.indexOf('-'))
+}
+
+release {
+    failOnPublishNeeded = false
+
+    buildTasks = []
```

Comment:
Need to add build config here https://github.com/ballerina-platform/module-ballerina-websub/pull/24/commits/3c9c29f14ef8c4428dabaf0b085b5ec9ff41b4b7